nahusenayElias/Team_4
# Initialize react frontend

## Related ticket
[SCRUM-4 React Frontend
](https://edu-team4-react24k.atlassian.net/browse/SCRUM-4?atlOrigin=eyJpIjoiNzUzMDMyMmNlZTg5NGZiNGIwNjlmZjY4N2ZmN2YwM2IiLCJwIjoiaiJ9)

## In this PR:
- initialized react frontend, basic page structure
- installed JSON:API Drupal module
- db dump has "frontpage" and "aboutpage" content types for testing fetching data from drupal to frontend

## Testing instructions:
- checkout branch
- run `lando start`
- run `lando db-import dump sql.gz`
- run `lando drush deploy` (combines drush cim and drush cr and some other useful commands)
- paste your localhost address that Drupal is running on to `frontend/src/services/api.jsx` "const localhostAddress"
- run `npm install` in frontend folder
- `npm run dev` to run frontend
- check that the "Home" and "About us" pages are getting data in from drupal
